repaygithub/ui-tools
# Cactus 617 dependencies

The major change to test is the upgrade to Webpack 5, which required a few minor config changes.  Otherwise, none of the changes appear to be big deals.

Also, I deleted the `create-repay-ui` package, as that's been replaced by the `ui-private` version.

### Testing

1. `cd modules/babel-preset && yalc publish`
2.  `cd modules/repay-scripts && yalc publish`
3. `cd modules/eslint-config && yalc publish`
4. `cd modules/testing-tools && yalc publish`
5. `cd` into cactus
6. `yalc add @repay/babel-preset && yalc add @repay/scripts && yalc add @repay/eslint-config && yalc add @repay/testing-tools`
7. Run `yarn` to update all transitive dependencies
8. Run `yarn build` to make sure everything still builds properly
9. Run `yarn w mock-ebpp start` and/or `yarn w standard start` to make sure those apps still work just fine.
10. Add `api.cache(true)` to the first line of the function body in the root Babel config file (this isn't a breaking change because Cactus is two versions behind...the breaking change came with the last upgrade).
11. Run `yarn lint`
12. Run `yalc w standard test:local` to make sure `@repay/testing-tools` still works.
